src/reducers/VaccineReducer.js

```diff
@@ -3,22 +3,25 @@ import { VACCINE_ACTIONS } from '../actions/VaccineActions';
 const initialState = () => ({
   isSyncingTemps: false,
   setLogIntervalFor: '',
+  error: null,
 });
 
 export const VaccineReducer = (state = initialState(), action) => {
   const { type } = action;
 
   switch (type) {
     case VACCINE_ACTIONS.DOWNLOAD_LOGS_START: {
-      return { ...state, isSyncingTemps: true };
+      return { ...state, isSyncingTemps: true, error: null };
     }
 
     case VACCINE_ACTIONS.DOWNLOAD_LOGS_ERROR: {
-      return { ...state, isSyncingTemps: false };
+      const { payload } = action;
+      const { error } = payload;
+      return { ...state, isSyncingTemps: false, error };
     }
 
     case VACCINE_ACTIONS.DOWNLOAD_LOGS_COMPLETE: {
-      return { ...state, isSyncingTemps: false };
+      return { ...state, isSyncingTemps: false, error: null };
```

Comment:
I agree - the hesitation was because I wanted to ask whether a single failure should be fatal or not. If one sensor fails to sync logs & info, should that abort the whole sync cycle?

Reply:
that's true - I think only when the whole process fails, rather than just an individual sensor? - I have added some state in this PR https://github.com/openmsupply/mobile/pull/3445/files in an effort to try and track individual sensor downloading errors.